zeppelin-jupyter/src/main/java/org/apache/zeppelin/jupyter/JupyterUtil.java

```diff
@@ -167,6 +164,100 @@ private Gson getGson(GsonBuilder gsonBuilder) {
         .registerTypeAdapterFactory(outputTypeFactory).create();
   }
 
+  public String getJson(String input) {
+    Note note = getNote(new StringReader(input), "%spark", "%md");
+    return new Gson().toJson(note);
+  }
+
+  public String getNbformat(String note) {
+    Note noteFormat = getGson(new GsonBuilder()).fromJson(note, Note.class);
+
+    JsonObject nbformat = new JsonObject();
+    JsonArray cells = new JsonArray();
+
+    RegularExpression MD = new RegularExpression("%md\\s");
+    RegularExpression SQL = new RegularExpression("%sql\\s");
+    RegularExpression UNKNOWN_MAGIC = new RegularExpression("%\\w+\\s");
+    RegularExpression HTML = new RegularExpression("%html\\s");
+    RegularExpression SPARK = new RegularExpression("%spark\\s");
+
+    int index = 0;
+    for (Paragraph paragraph : noteFormat.getParagraphs()) {
+      String code = StringUtils.stripStart(paragraph.getText(), " ");
+      JsonObject codeJson = new JsonObject();
+
+      if (code == null || code.trim().isEmpty())
+        continue;
+
+      System.out.println(code);
```

Comment:
why?

Reply:
Oops! I forgot to remove this line. Good catch